src/nvim/misc1.c

```diff
@@ -1617,30 +1606,23 @@ int del_chars(long count, int fixpos)
   return del_bytes(bytes, fixpos, TRUE);
 }
 
-/*
- * Delete "count" bytes under the cursor.
- * If "fixpos" is TRUE, don't leave the cursor on the NUL after the line.
- * Caller must have prepared for undo.
- *
- * return FAIL for failure, OK otherwise
- */
-int 
-del_bytes (
-    long count,
-    int fixpos_arg,
-    int use_delcombine                  /* 'delcombine' option applies */
-)
+/// Delete "count" bytes under the cursor.
+/// If "fixpos" is true, don't leave the cursor on the NUL after the line.
+/// Caller must have prepared for undo.
+///
+/// @param  count           number of bytes to be deleted
+/// @param  fixpos_arg      leave the cursor on the NUL after the line
+/// @param  use_delcombine  'delcombine' option applies
+///
+/// @return FAIL for failure, OK otherwise
+int del_bytes(colnr_T count, bool fixpos_arg, bool use_delcombine)
 {
-  char_u      *oldp, *newp;
-  colnr_T oldlen;
+  char_u *newp;
```

Comment:
minor nitpick: should either be the the line before/after `oldp` (or right before the `if (` where it gets assigned).